tests/test_Proxy.py

```diff
@@ -0,0 +1,306 @@
+import pytest
+import asyncio
+from starkware.starknet.testing.starknet import Starknet
+from utils import (
+    Signer, assert_revert, assert_event_emitted, get_contract_def, cached_contract
+)
+
+# contract paths
+account_path = 'contracts/Account.cairo'
+proxy_path = 'contracts/proxy/Proxy.cairo'
+dummy_path = 'tests/mocks/dummy_implementation.cairo'
+dummy_path_v2 = 'tests/mocks/dummy_implementation_v2.cairo'
```

Comment:
I think it'd be slightly more clear to just hardcode these where they're used, since they're not reused

Reply:
Good point